modules/cnv-deleting-vms.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-delete-vms.adoc
+
+[id="cnv-deleting-vms_{context}"]
+
+= Deleting a virtual machine and PVCs using the CLI
+
+When you delete a virtual machine, the PVC it uses is unbound. If you
+do not plan to bind this PVC to a different VM, delete it, too.
+
+You can only delete objects in the project you are currently working in,
+unless you specify the `-n <project_name>` option.
+
+.Procedure
+
+Run these commands to delete the virtual machine and the PVC.
+----
+$ oc delete vm fedora-vm
+----
+
+----
```

Comment:
But this is really two steps, so please separate the commands. You also need to explain the variables that you're using in the commands and how you know which PVC to delete.

Reply:
* Flagging how you find the correct PVC with QE.

Here is the topic now:


= Deleting a virtual machine and PVCs using the CLI

When you delete a virtual machine, the PVC it uses is unbound. If you
do not plan to bind this PVC to a different VM, delete it as well.

.Procedure

Run these commands to delete the virtual machine and the PVC.

NOTE: You can delete objects only in the project you are currently working in,
+unless you specify the `-n <project_name>` option, for project name.

. Run the following command to delete the virtual machine:
+
----
$ oc delete vm <fedora-vm>
----
+
. Run the following command to delete the PVC associated with the virtual machine.
+
----
$ oc delete pvc <fedora-vm-pvc>
----